offline-virtual-machine.md

```diff
@@ -0,0 +1,74 @@
+# The Offline Virtual Machine user guide
+
+If you require to get hold of non-running virtual machines, you will need to work
+with the OfflineVirtualMachine object. The OfflineVirtualMachine holds the
+template for a running VirtualMachine and holds additional metadata used
+for tracking in the KubeVirt ecosystem.
+
+The OfflineVirtualMachine work with the VirtualMachine, which is a running
+VirtualMachine, that is created upon user request for virtual machine to start.
+
+## When to use it
+
+Whenever you need to manipulate stopped virtual machine (changing/displaying
+configuration).
+
+## How to use it
+
+The OfflineVirtualMachine is designed as a [Kubernetes CRD](https://kubernetes.io/docs/concepts/api-extension/custom-resources/).
+CRD implies that you can use the OfflineVirtualMachine as you would use any
+other Kubernetes object.
+
+### Commandline
+
+Whenever you want to manipulate the OfflineVirtualMachine through the commandline
+you can use the kubectl command. The following are examples demonstrating how
+to do it.
+
+```bash
+# Define an OfflineVirtualMachine:
+kubectl create -f myofflinevm.yaml
+
+# Start an OfflineVirtualMachine:
```

Comment:
Will that start offline or livevm? Based on the above comments it looks like it will start an offlinevm and in my opinion we should be clear that it creates livevm.

Reply:
Well it will start the OfflineVirtualMachine by creating a VirtualMachine. I think there is no difference. There  exists nothing like "start an OfflineVirtualMachine now and start the corresponding VirtualMachine later".

@pkliczewski  Maybe rewriting it to "Start an OfflineVirtualMachine by creating the VirtualMachine object" removes the ambiguity?

Reply:
Naming it `StatefulVirtualMachine` insteasd of `OfflineVirtualMachine`  would probably have made that clearer.

Reply:
Maybe this can also be made clearer if the title is user focused (not technical): "# Start the virtual machine"

Reply:
@fabiand Your rewrite looks good to me